examples/cwt_sensor_connectivity.py

```diff
@@ -19,7 +20,12 @@
 from mne import io
 from mne_connectivity import spectral_connectivity_epochs, seed_target_indices
 from mne.datasets import sample
-from mne.time_frequency import AverageTFR
+
+# XXX: remove logic once support for mne<1.7 is dropped
+try:
+    from mne.time_frequency import AverageTFRArray as AverageTFR
+except ImportError:
+    from mne.time_frequency import AverageTFR
```

Comment:
hmm, I'm not crazy about including logic like this in a public-facing example/tutorial. Is it enough to code it whatever way works for the CI docs build, and include a comment to the effect of "if you're on some other version of MNE, use the other class"?

Reply:
Sure, I'm happy to take this approach as well.

Reply:
How do you feel about this?

https://github.com/mne-tools/mne-connectivity/blob/12f23ac888c9d55adaa3aa020b0af575ceebb733/examples/cwt_sensor_connectivity.py#L95-L96